yb-voyager/cmd/analyzeSchema.go

```diff
@@ -392,9 +392,11 @@ func checkStmtsUsingParser(sqlInfoArr []sqlInfo, fpath string, objType string) {
 			reportDeferrableConstraintCreateTable(createTableNode, sqlStmtInfo, fpath)
 			reportUnsupportedDatatypes(createTableNode.CreateStmt.Relation, createTableNode.CreateStmt.TableElts, sqlStmtInfo, fpath, objType)
 			parseColumnsWithUnsupportedIndexDatatypes(createTableNode)
+			reportUnsupportedConstraintsOnComplexDatatypesInCreate(createTableNode, sqlStmtInfo, fpath)
```

Comment:
Ideally, we should be adding code for new issues in queryissues 😬 
but this is urgent, so I think it's okay, we can take it up post release.

Reply:
Yeah, so for this one I realized at the very last minute that we will have to new issues of constraints and I had to move all the index reporting functions there as I am maintaining state of `compositeTypes` and `unsupportedDataTypesColumns` map then I will have to move all of that which will take time. So thought will take up later.